test/verify/check-packagekit

```diff
@@ -1460,14 +1466,16 @@ class TestAutoUpdates(NoSubManCase):
 
     def setUp(self):
         super().setUp()
-        # not implemented for apt yet, only dnf
-        self.supported_backend = "dnf" in self.backend
+        self.supported_backend = "dnf" in self.backend or self.backend == "apt"
```

Comment:
One missing converted test is `testWithAvailableUpdates` it now fails with `NotImplementedError`.